2020-04-30 00:24:57,558 ERROR 60633 [-/192.168.0.101/-/5ms GET /socket.io/?room=628754&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODgxNjk1MTYsImV4cCI6MTU4ODE5MTExNn0.V79TJ82uJ3ZQMk5prn5VXiBLkDjYyWFznatFHtxx4b0&EIO=3&transport=websocket] room service tick null
2020-04-30 00:25:31,431 ERROR 60633 [-/192.168.0.101/-/1ms GET /socket.io/?room=628754&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODgxNjk1MTYsImV4cCI6MTU4ODE5MTExNn0.V79TJ82uJ3ZQMk5prn5VXiBLkDjYyWFznatFHtxx4b0&EIO=3&transport=websocket] room service tick null
2020-04-30 00:29:30,694 ERROR 60633 [-/192.168.0.101/-/3ms GET /socket.io/?room=628754&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODgxNjk1MTYsImV4cCI6MTU4ODE5MTExNn0.V79TJ82uJ3ZQMk5prn5VXiBLkDjYyWFznatFHtxx4b0&EIO=3&transport=websocket] room service tick null
2020-04-30 00:30:16,787 ERROR 60633 [-/192.168.0.101/-/2ms GET /socket.io/?room=628754&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODgxNjk1MTYsImV4cCI6MTU4ODE5MTExNn0.V79TJ82uJ3ZQMk5prn5VXiBLkDjYyWFznatFHtxx4b0&EIO=3&transport=websocket] room service tick null
2020-04-30 00:30:16,970 ERROR 60633 [-/192.168.0.101/-/2ms GET /socket.io/?room=628754&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODgxNjk1MTYsImV4cCI6MTU4ODE5MTExNn0.V79TJ82uJ3ZQMk5prn5VXiBLkDjYyWFznatFHtxx4b0&EIO=3&transport=websocket] room service tick null
2020-04-30 00:41:54,185 ERROR 60633 [-/192.168.0.101/-/4ms GET /socket.io/?room=628754&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODgxNjk1MTYsImV4cCI6MTU4ODE5MTExNn0.V79TJ82uJ3ZQMk5prn5VXiBLkDjYyWFznatFHtxx4b0&EIO=3&transport=websocket] room service tick null
2020-04-30 00:42:28,928 ERROR 60633 [-/192.168.0.101/-/2ms GET /socket.io/?room=628754&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODgxNjk1MTYsImV4cCI6MTU4ODE5MTExNn0.V79TJ82uJ3ZQMk5prn5VXiBLkDjYyWFznatFHtxx4b0&EIO=3&transport=websocket] room service tick null
2020-04-30 00:43:02,652 ERROR 60633 [-/192.168.0.101/-/3ms GET /socket.io/?room=628754&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODgxNjk1MTYsImV4cCI6MTU4ODE5MTExNn0.V79TJ82uJ3ZQMk5prn5VXiBLkDjYyWFznatFHtxx4b0&EIO=3&transport=websocket] room service tick null
2020-04-30 00:54:28,717 ERROR 60633 [-/192.168.0.101/-/12ms GET /socket.io/?room=628754&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODgxNjk1MTYsImV4cCI6MTU4ODE5MTExNn0.V79TJ82uJ3ZQMk5prn5VXiBLkDjYyWFznatFHtxx4b0&EIO=3&transport=websocket] room service tick null
2020-04-30 01:00:54,475 ERROR 60633 [-/192.168.0.103/-/62ms GET /socket.io/?room=628754&token=undefined&EIO=3&transport=websocket] room service tick null
2020-04-30 01:02:44,104 ERROR 60633 [-/192.168.0.101/-/1ms GET /socket.io/?room=628754&token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7Im5pY2tOYW1lIjoiYyIsImFjY291bnQiOiJjYWkiLCJ1c2VySWQiOjF9LCJpYXQiOjE1ODgxNjk1MTYsImV4cCI6MTU4ODE5MTExNn0.V79TJ82uJ3ZQMk5prn5VXiBLkDjYyWFznatFHtxx4b0&EIO=3&transport=websocket] room service tick null
2020-04-30 01:03:02,178 ERROR 60633 [-/192.168.0.103/-/18ms GET /socket.io/?room=628754&token=undefined&EIO=3&transport=websocket] room service tick null
2020-04-30 01:35:35,358 ERROR 60632 nodejs.ClusterClientNoResponseError: client no response in 1854709ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 60632
hostname: 192.168.0.101

2020-04-30 02:13:38,297 ERROR 60632 nodejs.ClusterClientNoResponseError: client no response in 2242910ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 60632
hostname: 192.168.0.101

2020-04-30 02:51:15,067 ERROR 60632 nodejs.ClusterClientNoResponseError: client no response in 1956662ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 60632
hostname: 192.168.0.101

2020-04-30 04:05:22,137 ERROR 60632 nodejs.ClusterClientNoResponseError: client no response in 4406960ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 60632
hostname: 192.168.0.101

2020-04-30 05:53:34,096 ERROR 60632 nodejs.ClusterClientNoResponseError: client no response in 6451949ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 60632
hostname: 192.168.0.101

2020-04-30 07:41:45,992 ERROR 60632 nodejs.ClusterClientNoResponseError: client no response in 6451867ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 60632
hostname: 192.168.0.101

2020-04-30 08:17:43,444 ERROR 60632 nodejs.ClusterClientNoResponseError: client no response in 2137434ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 60632
hostname: 192.168.0.101

2020-05-03 14:23:37,336 ERROR 66993 [-/127.0.0.1/-/0ms GET /] nodejs.PROTOCOL_SEQUENCE_TIMEOUTError: Handshake inactivity timeout
    at Handshake.<anonymous> (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/protocol/Protocol.js:160:17)
    at Handshake.emit (events.js:311:20)
    at Handshake._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/protocol/sequences/Sequence.js:124:8)
    at Timer._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/protocol/Timer.js:32:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
    --------------------
    at Protocol._enqueue (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/jorky/code/TexasPokerGame/server/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/jorky/code/TexasPokerGame/server/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/jorky/code/TexasPokerGame/server/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/jorky/code/TexasPokerGame/server/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/jorky/code/TexasPokerGame/server/node_modules/ali-rds/node_modules/pify/index.js:29:7
    sql: select now() as currentTime;
code: "PROTOCOL_SEQUENCE_TIMEOUT"
fatal: true
timeout: 10000
name: "PROTOCOL_SEQUENCE_TIMEOUTError"
pid: 66993
hostname: 192.168.0.103

2020-05-03 16:03:23,831 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 5113874ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-03 17:51:27,296 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 6483435ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-03 19:39:30,738 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 6483387ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-03 22:21:24,118 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 9713320ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-03 23:30:13,548 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 4129426ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-03 23:35:20,795 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 306937ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-03 23:37:13,755 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 92134ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-03 23:39:05,676 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 90455ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 01:42:48,683 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 7218354ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 02:39:06,677 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 3377890ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 09:15:59,902 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 23813180ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 09:32:13,189 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 611375ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 10:01:02,017 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 1287981ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 10:19:54,009 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 892125ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 11:50:51,311 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 3316402ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 11:58:06,374 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 94949ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 13:45:20,062 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 5713446ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 15:12:50,566 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 5250467ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 18:25:39,406 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 10808539ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 20:13:43,838 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 6484153ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 22:01:49,616 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 6485742ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-04 23:49:55,744 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 6485561ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-05 01:38:01,620 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 6485833ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-05 02:54:21,630 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 4579717ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-05 03:26:32,624 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 1909340ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

2020-05-06 08:25:50,279 ERROR 67020 nodejs.ClusterClientNoResponseError: client no response in 104297559ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/jorky/code/TexasPokerGame/server/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 67020
hostname: 192.168.0.103

